broker/cloud_run/lvk/ps_to_storage/deploy.sh

```diff
@@ -0,0 +1,93 @@
+region="${4:-us-central1}"
+versiontag="${5:-v1_0}"
+# get the environment variable
+PROJECT_ID=$GOOGLE_CLOUD_PROJECT
+
+MODULE_NAME="alerts-to-storage"  # lower case required by cloud run
+ROUTE_RUN="/"  # url route that will trigger main.run()
+
+define_GCP_resources() {
+    local base_name="$1"
+    local separator="${2:--}"
+    local testid_suffix=""
+
+    if [ "$testid" != "False" ] && [ -n "$testid" ]; then
+        testid_suffix="${separator}${testid}"
+    fi
+    echo "${base_name}${testid_suffix}"
+}
+
+#--- GCP resources used in this script
+artifact_registry_repo=$(define_GCP_resources "${survey}-cloud-run-services")
+cr_module_name=$(define_GCP_resources "${survey}-${MODULE_NAME}")  # lower case required by cloud run
+gcs_json_bucket=$(define_GCP_resources "${PROJECT_ID}-${survey}_alerts")
+ps_deadletter_topic=$(define_GCP_resources "${survey}-deadletter")
+ps_input_subscrip=$(define_GCP_resources "${survey}-alerts_raw") # pub/sub subscription used to trigger cloud run module
+ps_topic_alert_in_bucket=$(define_GCP_resources "projects/${PROJECT_ID}/topics/${survey}-alert_in_bucket")
+ps_trigger_topic=$(define_GCP_resources "${survey}-alerts_raw")
+runinvoker_svcact="cloud-run-invoker@${PROJECT_ID}.iam.gserviceaccount.com"
+
+if [ "${teardown}" = "True" ]; then
+    # ensure that we do not teardown production resources
+    if [ "${testid}" != "False" ]; then
+        echo
+        echo "Deleting resources for ${MODULE_NAME} module..."
+        gsutil rm -r "gs://${gcs_json_bucket}"
+        gcloud pubsub subscriptions delete "${ps_input_subscrip}"
+        gcloud pubsub topics delete "${ps_topic_alert_in_bucket}"
+        gcloud run services delete "${cr_module_name}" --region "${region}"
+    fi
+else
+    echo
+    echo "Creating gcs_json_bucket and uploading files..."
+    if ! gsutil ls -b "gs://${gcs_json_bucket}" >/dev/null 2>&1; then
+        gsutil mb -b on -l "${region}" "gs://${gcs_json_bucket}"
+        gsutil uniformbucketlevelaccess set on "gs://${gcs_json_bucket}"
+        gsutil requesterpays set on "gs://${gcs_json_bucket}"
+        gcloud storage buckets add-iam-policy-binding "gs://${gcs_json_bucket}" \
+            --member="allUsers" \
+            --role="roles/storage.objectViewer"
+    else
+        echo "${gcs_json_bucket} already exists."
+    fi
+
+    echo
+    echo "Configuring Pub/Sub notifications on GCS bucket..."
+    trigger_event=OBJECT_FINALIZE
+    format=json  # json or none; if json, file metadata sent in message body
+    gsutil notification create \
+        -t "$ps_topic_alert_in_bucket" \
+        -e "$trigger_event" \
+        -f "$format" \
+        "gs://${gcs_json_bucket}"
+
+    #--- Deploy Cloud Run service
+    echo
+    echo "Creating container image for ${MODULE_NAME} module and deploying to Cloud Run..."
+    moduledir="."  # assumes deploying what's in our current directory
+    config="${moduledir}/cloudbuild.yaml"
+    url=$(gcloud builds submit --config="${config}" \
```

Comment:
```suggestion
    # Deploy the service and capture the endpoint's URL.
    url=$(gcloud builds submit --config="${config}" \
```

Since this line starts with `url=`, the fact that this line actually deploys the service is obscured, which has tripped me up a couple times.